src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { PropsWithChildren, useState } from "react";
+
+import { FaMinus } from "@react-icons/all-files/fa/FaMinus";
+import { FaPlus } from "@react-icons/all-files/fa/FaPlus";
+import IconButton from "../IconButton";
+import { titleRow } from "./Accordion.module.scss";
+
+interface AccordionProps {
+  title: string | JSX.Element;
+}
+
+export default function Accordion({
+  title,
+  children,
+}: PropsWithChildren<AccordionProps>) {
+  const [open, setOpen] = useState(false);
+  return (
+    <div id="faq">
+      <a className={titleRow} onClick={() => setOpen(!open)}>
```

Comment:
Hmm I can see the rationale on why this can be an anchor... But i think for our use case, a button is more appropriate because we don't have any url changes or what not